src/components/Card/card.module.scss

```diff
@@ -0,0 +1,130 @@
+.card {
+    --bg-color: white;
+
+    box-shadow: $shadow-object-m;
+    background-color: var(--bg-color);
+    border-radius: $corner-radius-xl;
+    color: black;
+    display: block;
+    font-family: $octuple-font-family;
+    gap: $space-m;
+    padding: $space-ml;
+    width: 360px;
+    position: relative;
+
+    &.carousel {
+        border: 1px solid var(--grey-color-20);
+    }
+
+    .icon {
+        color: inherit;
+    }
+
+    .message {
+        flex: 1;
+        display: flex;
+        align-items: center;
+        color: inherit;
+    }
+
+    .buttons-container {
+        padding: $space-m;
+    }
+
+    &-large {
+        height: 278px;
+    }
+
+    &-medium,
+    &-small {
+        height: 260px;
+    }
```

Comment:
wondering how we can leave this to implementation to decided. These are overly prescriptive